docs/_documentations/troubleshooting.md

```diff
@@ -306,7 +242,96 @@ If you have the Codewind Language Server for Node.js Profiling extension enabled
+
+<!--
+Action/Topic: Creating a project and/or Checking the application and build statuses
+Issue type: bug/info
+Codewind version: 0.5.0
+OS: Windows
+IDE extension version: 0.5.0
+IDE version: Eclipse 2019-09
+Issue link: https://github.com/eclipse/codewind/issues/715
+-->
+## Projects stuck in starting or stopped state
+You might occasionally see projects stuck in the `Starting` or `Stopped` state even though the container logs say the projects are up and running. This can happen when you create a number of projects, for example, using the default and Appsody templates with Codewind 0.5.0. 
+
+**Workaround** Manually rebuild the projects that are stuck in `Starting` or `Stopped` state. To do this: 
+1. In the **Codewind Explorer** view, right-click your project and select **Build**.
+2. Wait for the project state to return to **Running** or **Debugging** in the **Codewind Explorer** view.
+
+<!--
+Action/Topic: Checking the application status 
+Issue type: bug/info
+Codewind version: 0.9.0
+Issue link: https://github.com/eclipse/codewind/issues/1269
+-->
+## Application stuck in Starting state 
+Some project templates come with no server configured by default, like Appsody Node.js. The application status cannot be determined for these types of projects because Codewind relies on application endpoints for status. Codewind determines the port of the application by inspecting the project's container information. The container may have a port exposed but since no server is available to ping on that port, the status check times out and the state is stuck on `Starting`. 
+
+**Workaround** This is not inherently a problem but you can disable the status check for the project by taking the following steps:
+1. Edit the `.cw-settings` file under the application, and set the key `internalPort` to `-1`. 
+    - This forces the project state to `Stopped`, stops pinging the project's health check endpoint, and ignores the timeout error. 
+2. Once you implement the project's server, revert the setting by setting `internalPort` to `""` to allow Codewind to use the default port of the container. Alternatively, choose a specific port if your container exposes multiple ports. 
+
+<!--
+Action/Topic: Checking the application status 
+Issue type: bug/info
+Codewind version: 0.9.0
+Issue link: https://github.com/eclipse/codewind/issues/1269
+-->
+## How to create a .cw-settings file if it does not exist
+Prior to the 0.9.0 release, some Appsody and OpenShift Do (odo) templates did not come with a `.cw-settings`. The `.cw-settings` file usually created automatically. It tells Codewind how to interact with a project. Things like application and build status are tied to these settings. If you have a project from those stacks from a previous release you need to create the .cw-settings file manually. The file must reside under the project root directory. 
```

Comment:
...The `.cw-settings` file usually creates automatically and tells Codewind how to interact with a project...If you have a project from those stacks from a previous release, you must manually create the .cw-settings file. The file must reside under the project root directory.

Reply:
Fixed most of it but thought `file usually creates automatically` sounded like the file is doing something? It's now `The `.cw-settings` file is usually created automatically and tells Codewind how to interact with a project.` Is that okay?